.github/SECURITY.md

```diff
@@ -15,23 +15,15 @@ Versions of AzerothCore:
 | AzerothCore Branch | Supported          |
 | ------------------ | ------------------ |
 | master             | :white_check_mark: |
+| Any playerbot fork | :red_circle:       |
```

Comment:
May also want to mention NPC bots since they're a separate fork as well.

Reply:
`Any bot fork`

Reply:
I will update once CI is done, dont want to interrupt it for trivial changes as I am running everything